api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -50,7 +50,7 @@
 	        + "/condition-ver-status";
 	
 	public static final String CLINICAL_FINDINGS_SYSTEM_URI = HL7_FHIR_CODE_SYSTEM_PREFIX + "/clinical-findings";
-
+	
```

Comment:
Can we exclude this change from this PR? It prevents it from interfering with other PRs that might make the same change.

Reply:
@ibacher Yes definitely. I will remove it.